geoschem.go

```diff
@@ -47,8 +47,12 @@ const (
 type GEOSChem struct {
 	aVOC, bVOC, aSOA, bSOA, nox, no, no2, pNO, sox, pS, nh3, pNH, totalPM25 map[string]float64
 
+	noHour bool
```

Comment:
Change to `noChemHour` because there are other types of input files that do have hour indexes